inspectit-oce-core/src/main/java/rocks/inspectit/oce/core/instrumentation/hook/DispatchHookAdvice.java

```diff
@@ -14,8 +14,8 @@
      * The content of this method is added before the target method! See {@link Advice} for existing parameter annotations.
      */
     @Advice.OnMethodEnter
-    public static void onEnter(@Advice.Origin Object origin) {
-        System.out.println("\tEntering: " + origin);
+    public static void onEnter(@Advice.Origin Object origin, @Advice.This Object thiz) {
```

Comment:
this still also works on static methods, right?

Reply:
No it doesn't :) this was a test and shouldn't be checked in